packages/dbos-kafkajs/index.ts

```diff
@@ -11,7 +11,7 @@ import {
   logLevel,
   Partitioners,
 } from 'kafkajs';
-import { DBOS, Step, StepContext, ConfiguredInstance, DBOSContext, DBOSEventReceiver } from '@dbos-inc/dbos-sdk';
+import { DBOS, ConfiguredInstance, DBOSEventReceiver } from '@dbos-inc/dbos-sdk';
```

Comment:
Feel free to break anything you want in this package and/or the Confluent Kafka package. I'll be replacing both